sources/packages/web/src/components/common/Reports.vue

```diff
@@ -51,6 +51,8 @@ export default defineComponent({
     const REPORT_TYPE_DROPDOWN_KEY = "reportName";
     const loading = ref(false);
     const PROGRAM_YEAR_DROPDOWN_KEY = "programYear";
+    const INSTITUTION_NAME = "institutionName";
```

Comment:
Since it is a dropdown, it should follow the same naming conversion from `REPORT_TYPE_DROPDOWN_KEY` and `PROGRAM_YEAR_DROPDOWN_KEY` which would be `INSTITUTION_NAME_DROPDOWN_KEY`.